book/src/installers/customizing.md

```diff
@@ -0,0 +1,109 @@
+these users, our installers allow several inputs which give the user more
+control over their installation experience via configuration. Most of these are
+implemented using environment variables, however several are also available as
+arguments to the installer scripts themselves.
+
+Several of these options were available in earlier releases of `dist`, but they
+are formally stabilized as of `0.23.0`.
+
+## Install location
+
+One of the most important jobs of an installer is to place the necessary
+artifacts in the correct location on the target machine. `dist` allows you to
+configure this for your particular needs. Depending on your setup, the following
+options are available for end users to further configure this location:
+
+- `${app name}_INSTALL_DIR`
+
+    > Replace `{app name}` with the name of the application. To transform the
+    > app name to the env var, replace any spaces or hyphens with an underscore
+    > and then put it in all caps. You can double check this transform by
+    > comparing the `install_dir_env_var` value in your `dist-manifest.json`.
+
+    This environment variable tells the installer what to use as the "base"
+    directory for the installation task. This may not necessarily be the exact
+    directory your binaries are installed to based on your install layout. For
+    example, this directory may have `./bin` appended to it.
+
+    This environment variable is also known as `CARGO_DIST_FORCE_INSTALL_DIR`. 
+
+- `$HOME`
+
+    This environment variable is sourced if you use the `CARGO_HOME` or
+    `~/sub/dir` install location options.
+
+- `${custom env var}`
+
+    You can use a custom environment variable to specifying your install
+    location. If you do, that variable will be sourced during the install task.
+
+- `$CARGO_HOME`
+
+    This environment variable is sourced if you use the `CARGO_HOME` install
+    location.
+
+## `$PATH`
+
+`$PATH` is an environment variable that pre-exists on nearly all systems and
+lists locations to look for executables in. This is what allows you to call a
+program by its name instead of needing to call it by it's specific location on
+the file system.
+
+When you use axo installers, we do the heavy lifting of ensuring that your
+application is available "on PATH" to your end users. However, there are
+circumstances where this is not desirable, and so we provide the ability to
+skip this setup step.
+
+- ### `$PATH`
+
+    This environment variable is both sourced and modified in the standard mode
+    of an installation. We source this variable to see if the installation
+    location is already on PATH and we will modify it if it is not.
+
+- ### `$PATH` modification
+
+    If you do not want your PATH to be modified you can use the following to
+    configure your installation experience:
+
+    - Env var: `$INSTALLER_NO_MODIFY_PATH=1` for all installers
+    - Shell argument: `--no-modify-path`
```

Comment:
We're in the process of deprecating the commandline switches and moving towards the environment variable; I'd recommend not documenting these.

Reply:
ack